extensions/wikia/AdEngine/js/ml/modelFactory.js

```diff
@@ -11,6 +11,7 @@ define('ext.wikia.adEngine.ml.modelFactory', [
 		'name',
 		'enabled'
 	];
+	var qs = new Querystring();
```

Comment:
If `create()` will be called only once, isn't it better idea to place it there? Correct me if I'm wrong but `qs` object will be created on every pageview no matter if it will or won't execute model, right?

Reply:
Good point.

Reply:
+1 it has to be done in `create` because of mobile-wiki